pkg/actions/webhook.go

```diff
@@ -147,6 +147,7 @@ func executeAndLogResponse(ctx context.Context, req *http.Request, buf *bytes.Bu
 	elapsed := time.Since(start)
 	buf.WriteString(fmt.Sprintf("\nRequest duration: %s\n", elapsed))
 	if err != nil {
+		buf.WriteString(fmt.Sprintf("\nError: %s\n", err))
```

Comment:
What happens if it fail on line 85?

Reply:
In that case we didn't even try to execute the URL.
I suggest creating a new method "logRunError" which will just write error to buffer and close it.
In that case we might want to map all possible error types and format them properly

Reply:
- Think we can long the `err` of the failed task at the `service.go` (runTasks) level.
- We can use `fmt.Fprintf(buf, "\nError: %s\n", err)` instead of the combination of `Sprintf` and `WriteString`